### ISSUE TITLE
Making Memes Accessible

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#93]
### Links
- Paper: https://dl.acm.org/citation.cfm?id=3353792
### Abstract
- Images on social media platforms are inaccessible to people with vision impairments due to a lack of descriptions that can be read by screen readers. Providing accurate alternative text for all visual content on social media is not yet feasible, but certain subsets of images, such as internet memes, offer affordances for automatic or semi-automatic generation of alternative text. We present two methods for making memes accessible semi-automatically through (1) the generation of rich alternative text descriptions and (2) the creation of audio macro memes. Meme authors create alternative text templates or audio meme templates, and insert placeholders instead of the meme text. When a meme with the same image is encountered again, it is automatically recognized from a database of meme templates. Text is then extracted and either inserted into the alternative text template or rendered in the audio template using text-to-speech. In our evaluation of meme formats with 10 Twitter users with vision impairments, we found that most users preferred alternative text memes because the description of the visual content conveys the emotional tone of the character. As the preexisting templates can be automatically matched to memes using the same visual image, this combined approach can make a large subset of images on the web accessible, while preserving the emotion and tone inherent in the image memes.

- ミーム（Meme: 文字が加えられた画像）を視覚障害者向けにアクセシブルにするシステムを提案

<img width="447" alt="スクリーンショット 2019-10-31 16 44 54" src="https://user-images.githubusercontent.com/26996041/67928207-cb712b80-fbfd-11e9-8fc0-0f85bac0a695.png">

<img width="427" alt="スクリーンショット 2019-10-31 16 42 58" src="https://user-images.githubusercontent.com/26996041/67928179-bf856980-fbfd-11e9-9f56-320e75dc7292.png">
